Dockerfile

```diff
@@ -1,19 +1,25 @@
 # vim:set ft=dockerfile:
 FROM condaforge/mambaforge
-MAINTAINER https://github.com/bird-house/finch
+ARG DEBIAN_FRONTEND=noninteractive
+ENV PIP_ROOT_USER_ACTION=ignore
+LABEL org.opencontainers.image.authors="bourgault.pascal@ouranos.ca"
 LABEL Description="Finch WPS" Vendor="Birdhouse" Version="0.11.1"
 
+# Switch to /code directory
 WORKDIR /code
+COPY . ./
 
+# Build finch environment
 COPY environment.yml .
-RUN mamba env create -n finch -f environment.yml \
-    && mamba install -n finch gunicorn psycopg2 \
-    && mamba clean -ay
-
-COPY . .
```

Comment:
Keep this `COPY` below all the mamba install for layer re-use.  In a Dockerfile, we want the most frequently changed at the bottom for layer re-use to speed up and minimize space use during dev builds.